enterprise/plugins/com.liferay.ide.kaleo.core/src/com/liferay/ide/kaleo/core/KaleoCore.java

```diff
@@ -163,7 +163,7 @@ public static void updateKaleoConnectionSettings( ILiferayServer server )
     public static void updateKaleoConnectionSettings( ILiferayServer server, IKaleoConnection connection )
     {
         connection.setHost( server.getHost() );
-        //connection.setHttpPort( server.getHttpPort() );
```

Comment:
instead we should fix getHttpPort() method for DXP bundles instead of hard-coding this one.